guides/bookshop-astro-guide/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+_schema: guide-item
+_uuid: 12cbb415-2770-4302-b97d-9cb5c939f3ad
+_created_at: '2023-02-01 02:40:30 +0000'
+title: Introduction to Bookshop with Astro
+nav_title: Introduction to Bookshop with Astro
+order: 1
+published: true
+image: /documentation/static/bookshop-11ty/bookshop-astro-og.png
+description: >-
+  Learn how to build a live-editable website using Astro and Bookshop on
+  CloudCannon.
+tags: []
+related_articles:
+related_links: []
+guide_listing:
+  show_on_index: true
+  category: Bookshop Guides
+---
+Welcome to the guide for setting up an Astro website using Bookshop. Through this guide, we'll cover what Bookshop is and what it brings to the table, how to integrate it into an Astro website, and how to connect everything together to produce a fully static website that can be edited live in the browser using CloudCannon.
+
+## What is Bookshop?
+
+Bookshop is a collection of tooling that provides a component development workflow for static websites, and aids in the creation of a page-building interface in the CloudCannon CMS.
+
+Bookshop is fully open-source and is available on GitHub at [CloudCannon/bookshop](https://github.com/CloudCannon/bookshop/). As a tool that integrates into your codebase, we want to ensure you aren't vendor-locked to our platform. Sites built using Bookshop remain fully portable and can be built or hosted anywhere on the web.
+
+## What does Bookshop bring to Astro?
+
+For developers working locally, Bookshop first provides an Astro integration that is used to live update your components. When dealing with your components, this integration provides a `bookshop:live` directive to mark a component for live editing. The `bookshop:live` directive can be added to any component referenced from an Astro template. This looks like:
+
+<comp.CodeBlock language="astro" source="src/pages/index.astro">
+```
+<Hello bookshop:live text="My title text" />
+```
+</comp.CodeBlock>
+
+We'll cover more about components in another chapter; for now, the important thing to know is that Bookshop components can be written using either Astro or React and, in most cases, can directly use your existing templating.
```

Comment:
Could you also use write bookshop components with Svelte and use them with Astro?

Reply:
🙅‍♂️